debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java

```diff
@@ -661,10 +662,16 @@ public static String parseSetAndEnumOptions(String typeExpression) {
             String literals = matcher.group(2);
             Matcher optionMatcher = ENUM_AND_SET_OPTIONS.matcher(literals);
             StringBuilder sb = new StringBuilder();
+            boolean first = true;
             while (optionMatcher.find()) {
                 String option = optionMatcher.group(1);
                 if (option.length() > 0) {
-                    sb.append(option.charAt(0));
+                    if (first) {
+                        first = false;
+                    } else {
+                        sb.append(ENUM_AND_SET_DELIMINATOR);
+                    }
+                    sb.append(option);
                 }
             }
             return sb.toString();
```

Comment:
Actually my concern was suppose we had column which can  have maximum 50/60 enum values, schema builder will always contain it as comma separated string, then 
1. for each row insert, update i will be **un necessary**  creating array of 60 strings values just to get **single enum value** at particular index . 
2. similarly i will be performing **split based on deliminator** for each row insert/update.

Above two can be **high performance** impact, if my number of row entries are huge.